src/js/app.js

```
@@ -1206,6 +1222,14 @@ function attachOrgCardListeners(root) {
     img.__attached = true;
   });
 
+  root.querySelectorAll("img.org-logo").forEach((img) => {
+    observeImage(img);
```

ESLint

[error] 'observeImage' is not defined.

(no-undef)